.clang-format

```diff
@@ -112,5 +114,6 @@ SpacesInSquareBrackets: true
 Standard:        Cpp11
 TabWidth:        4
 UseTab:          Never
+UseCRLF:         Always
```

Comment:
Doesn't work with clang-format 12:
```
c:\repositories\qskinny_vrcomputing\.clang-format:117:18: error: invalid boolean
UseCRLF:         Always
                 ^~~~~~
Error reading c:\repositories\qskinny_vrcomputing\.clang-format: invalid argument
```

(deprecated) see: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#usecrlf

upgrade to clang-format 16's

see: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#lineending

alternatively 

```
UseCRLF:         true
```